app/src/main/AndroidManifest.xml

```diff
@@ -38,6 +38,16 @@
                 android:value="" />
         </activity>
 
+        <activity
+            android:name=".ui.screens.newsbrainz.NewsBrainzActivity"
+            android:exported="false"
+            android:label="@string/title_activity_news_brainz"
+            android:theme="@style/AppTheme">
```

Comment:
The default theme for the app is this, hence even if we don't duplicate it here, it will pick the same theme.